### DISCUSSION TITLE
Form not rendering with initial values after http load

### DISCUSSION THREAD

**Discussion by viveleroi** [jaredpalmer/formik#3239] [Q&A] [answered]
I'm loading data for a form from our API and passing it into my `Form` component as a prop. I can clearly see the right data via console.log but the form is never updating with those initialValues.

When I hard-code a value, it works. I don't understand what I've missed. The form component never shows the data.

https://paste.gg/p/anonymous/e0737684676a4df5b7e0a537d179a752

**Comment by johnrom** [comment, marked as answer]
You have to set `enableReinitialize` on the Formik component to enable updates via initialValues
